tests/testData.json

```diff
@@ -0,0 +1,141 @@
+{
+	"com.github.vilnius-leopold.node-gsettings.testing": [
+		{
+			"key": "boolean-setting",
+			"originalValue": false,
+			"testValues": [
+				true
+			],
+			"failTestValues": [
+				null,
+				"string"
+			]
+		},
+		{
+			"key": "unsigned-integer-setting",
+			"originalValue": 15,
+			"testValues": [
+				9134534,
+				0,
+				4294967295
+			],
+			"failTestValues": [
+				null,
+				-1,
+				0.2,
+				4294967296
+			]
+		},
+		{
+			"key": "integer-setting",
+			"originalValue": -15,
+			"testValues": [
+				2147483647,
+				-2147483648,
+				2312,
+				-3423
+			],
+			"failTestValues": [
+				"some sting",
+				null,
+				["an array", 12, "stuff"],
+				0.12,
+				2147483648,
+				-2147483649
+			]
+		},
+		{
+			"key": "integer-range-setting",
+			"originalValue": -2,
+			"testValues": [
+				0,
+				2,
+				10,
+				-10
+			],
+			"failTestValues": [
+				-11,
+				11
+			]
+		},
+		{
+			"key": "unsigned-integer-range-setting",
+			"originalValue": 10,
+			"testValues": [
+				5,
+				15
+			],
+			"failTestValues": [
+				4,
+				16
+			]
+		},
+		{
+			"key": "double-range-setting",
+			"originalValue": 0.0,
+			"testValues": [
+				5.05,
+				0.002,
+				-5.05
+			],
+			"failTestValues": [
+				5.05000001,
+				-5.05000001
+			]
+		},
+		{
+			"key": "double-setting",
+			"originalValue": 5,
+			"testValues": [
+				-12.346,
+				6.9999999999,
+				179769313486231570814527423731704356798070567525844996598917476803157260780028538760589558632766878171540458953514382464234321326889464182768467546703537516986049910576551282076245490090389328944075868508455133942304583236903222948165808559332123348274797826204144723168738177180919299881250404026184124858368.000000,
```

Comment:
doesn't this `1.7976931348623157e+308` neglect diggists? The reason why I use the expanded form is to make sure that no strange things happen when reaching the MAX/MIN boundries. Meaning +1/-1 has to be the difference between fail and success. ...I'm I getting this?